src/Utilities/Compiler/WordParser.cs

```diff
@@ -98,7 +99,7 @@ public WordParser(string text, WordParserOptions options, char prefix)
         {
             if (options < WordParserOptions.None || options > (WordParserOptions.IgnoreMnemonicsIndicators | WordParserOptions.SplitCompoundWords))
             {
-                throw new ArgumentException($"'{nameof(options)}' ({((int)options).ToString()}) is invalid for Enum type'{typeof(WordParserOptions).Name}'");
+                throw new ArgumentException($"'{nameof(options)}' ({((int)options).ToString(CultureInfo.InvariantCulture)}) is invalid for Enum type'{typeof(WordParserOptions).Name}'");
```

Comment:
😕 This seems unnecessarily complex

```suggestion
                throw new ArgumentException($"'{(int)options}' is invalid for enum type '{typeof(WordParserOptions).Name}'", nameof(options));
```